website/docs/r/emr_cluster.html.markdown

```diff
@@ -0,0 +1,316 @@
+* `charge_type` - (Required, ForceNew) Charge Type for this cluster. Supported value: PostPaid or PrePaid. Default value: PostPaid.
+* `zone_id` - (Required, ForceNew) Zone ID, e.g. cn-huhehaote-a
+* `security_group_id` (Optional, ForceNew) Security Group ID for Cluster, you can also specify this key for each host group.
+* `vswitch_id` (Optional, ForceNew) Global vswitch id, you can also specify it in host group.
+* `option_software_list` (Optional, ForceNew) Optional software list.
+* `high_availability_enable` (Optional, ForceNew) High Available for HDFS and YARN. If this is set true, MASTER group must have two nodes.
+* `use_local_metadb` (Optional, ForceNew) Use local metadb. Default is false.
+* `ssh_enable` (Optional, ForceNew) If this is set true, we can ssh into cluster. Default value is false.
+* `master_pwd` (Optional, ForceNew) Master ssh password.
+* `eas_enable` (Optional, ForceNew) High security cluster (true) or not. Default value is false.
+* `user_defined_emr_ecs_role` (Optional, ForceNew) Alicloud EMR uses roles to perform actions on your behalf when provisioning cluster resources, running applications, dynamically scaling resources. EMR uses the following roles when interacting with other Alicloud services. Default value is AliyunEmrEcsDefaultRole.
+* `key_pair_name` (Optional, ForceNew) Ssh key pair.
+* `deposit_type` (Optional, ForceNew) Cluster deposit type, HALF_MANAGED or FULL_MANAGED.
+* `related_cluster_id` (Optional, ForceNew) This specify the related cluster id, if this cluster is a Gateway.
+* `host_group` - (Optional) Groups of Host, You can specify MASTER as a group, CORE as a group (just like the above example).
+
+#### Block host_group
+The host_group mapping supports the following: 
+* `host_group_name` - (Required, ForceNew) host group name.
+* `host_group_type` - (Required) host group type, supported value: MASTER, CORE or TASK.
+* `charge_type` - (Optional) Charge Type for this group of hosts: PostPaid or PrePaid. If this is not specified, charge type will follow global charge_type value.
+* `period` - (Optional) If charge type is PrePaid, this should be specified, unit is month. Supported value: 1、2、3、4、5、6、7、8、9、12、24、36.
+* `node_count` - (Required) Host number in this group.
+* `instance_type` - (Required) Host Ecs instance type.
+* `disk_type` - (Required) Data disk type. Supported value: cloud,cloud_efficiency,cloud_ssd,local_disk,cloud_essd.
+* `disk_capacity` - (Required) Data disk capacity.
+* `disk_count` - (Required) Data disk count.
+* `sys_disk_type` - (Required) System disk type. Supported value: cloud,cloud_efficiency,cloud_ssd,cloud_essd.
+* `sys_disk_capacity` - (Required) System disk capacity.
+* `auto_renew` - (Optional) Auto renew for prepaid, true of false. Default is false.
+* `instance_list` - (Optional) Instance list for cluster scale down. This value follows the json format, e.g. ["instance_id1","instance_id2"]. escape character for " is \".
+  
```

Comment:
If the resource supports self-define timeouts, please show them, like https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/website/docs/r/instance.html.markdown#timeouts